src/constants/token/pancakeswap.json

```
@@ -438,6 +438,16 @@
       "address": "0x1A2fb0Af670D0234c2857FaD35b789F8Cb725584",
       "chainId": 56,
       "decimals": 18
+    },
+    {
+      "name": "CROCODILE",
+      "symbol": "CRO",
+      "address": "0x1e441f85F52B6D680E4C6ee7d969a9FF4465Fc69",
+      "chainId": 56,
+      "decimals": 18
```

Think you're missing a comma here

```suggestion
      "decimals": 18,
```

---

Okey sir